performance_tests.py

```
@@ -7,308 +6,340 @@
         connection_times = []
-        
+
         for i in range(iterations):
             start_time = time.time()
-            
+
             client = AsyncPocketOptionClient(
-                session_id=self.session_id,
-                is_demo=self.is_demo
+                session_id=self.session_id, is_demo=self.is_demo
             )
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'session_id' in constructor call

(E1123)